integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/stepDefinitions/APISteps.java

```diff
@@ -13,25 +13,23 @@
 import java.net.URISyntaxException;
 import java.net.http.HttpResponse;
 import java.util.Map;
-import java.util.Optional;
 import java.util.UUID;
+import java.util.logging.Logger;
 
 import static gov.uk.di.ipv.cri.common.api.util.IpvCoreStubUtil.sendCreateAuthCodeRequest;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 public class APISteps {
-
+    private static final Logger LOG = Logger.getLogger(APISteps.class.getName());
     private static final String ENVIRONMENT = "/dev"; // dev, build, staging, integration
     private static String devSessionUri;
     private static String devAuthorizationUri;
     public static String devAccessTokenUri;
     private static final ObjectMapper objectMapper = new ObjectMapper();
     private static final String DEFAULT_REDIRECT_URI =
-            Optional.ofNullable(System.getenv("IPV_CORE_STUB_URL") + "/callback")
-                    .orElse("https://di-ipv-core-stub.london.cloudapps.digital/callback");
-    private static final String DEFAULT_CLIENT_ID =
-            Optional.ofNullable(System.getenv("DEFAULT_CLIENT_ID")).orElse("ipv-core-stub");
```

Comment:
We shouldn't do this yet, it save to leave this till eventual PAAS decommissioning, right now changes so far are good with KBV & ADDRESS other CRI's haven't caught up yet.

Reply:
Actually Ola, `Optional.ofNullable(System.getenv("IPV_CORE_STUB_URL") + "/callback").orElse("https://di-ipv-core-stub.london.cloudapps.digital/callback");` will become "/callback" if `IPV_CORE_STUB_URL` env is not present. The behaviour we want of it using the default value never have worked as  `System.getenv` will never return `NULL` even if the environment variable is missing. Hence this update. 

You can test this by doing:

`String val = System.getenv("someRandomEnvThatDoesNotExit");`
You can observe that `val` in this example is not null and has the value `""`

Reply:
What we could do instead would be something like:

`private static final String DEFAULT_REDIRECT_URI = System.getenv("IPV_CORE_STUB_URL").length() > 0 
  ? (System.getenv("IPV_CORE_STUB_URL")+ "/callback") 
  : "https://di-ipv-core-stub.london.cloudapps.digital/callback";`